index.html

```diff
@@ -6,10 +6,12 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="shortcut icon" type="image/png" href="img/logo.png" />
 
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
```

Comment:
Why did you add these again?

Reply:
I had been using a wrong font that is not in the specification. I changed to the specified font sorry.